react/features/prejoin/components/DropdownButton.tsx

```diff
@@ -1,16 +1,11 @@
 import { Theme } from '@mui/material';
-import { withStyles } from '@mui/styles';
 import React from 'react';
+import { makeStyles } from 'tss-react/mui';
 
 import Icon from '../../base/icons/components/Icon';
 
 type Props = {
```

Comment:
maybe change this to interface as well if this is the plan.

Reply:
Will do in a future PR